src/Hazelcast.Net/Core/ClientVersion.cs

```diff
@@ -23,7 +23,7 @@ namespace Hazelcast.Core
     internal static class ClientVersion
     {
         private static string _clientVersion;
-        private static string _clientVersionPure;
+        private static string _clientMajorMinorPatchVersion;
```

Comment:
For this class, should we rather port the corresponding Java class [Version.java](https://github.com/hazelcast/hazelcast-mono/blob/master/hazelcast/hazelcast/src/main/java/com/hazelcast/version/Version.java) so that the two code base uses same logics?

Reply:
I have used the canonical form of the client version. Which prints `5.5.0+1ce4f2`. Is that OK for you?

Reply:
should we not include the commit id to be similar to what java client reports?

Reply:
Java reports the whole version info except the building part, such as `6.0.0-SNAPSHOT`. I've refactor based on this. You can see that Authenticator and ClientMetricSource sends the client version similar to Java client.